tests/helpers/test_plates_helper.py

```diff
@@ -98,31 +121,33 @@ def test_get_wells_for_destination_plate_successful(
                 "rna_id": "RNA-S-00001-00000001",
                 "lab_id": "MK",
                 "lh_sample_uuid": "499324b8-e4cc-4baf-a1a5-86a3a449b573",
+                "picked": True,
+                "created_at": timestamp,
+                "date_picked": timestamp,
             },
             {
-                "automation_system_run_id": "1",
+                "automation_system_run_id": 1,
                 "destination_coordinate": "A1",
-                "type": "sample",
-                "source_barcode": "DS000010001",
-                "source_coordinate": "B2",
-                "rna_id": "RNA-S-00001-00000009",
-                "lab_id": "MK",
-                "lh_sample_uuid": "c068cfea-02be-4aec-a50d-631abcd34573",
-            },
-            {
-                "automation_system_run_id": "1",
-                "destination_coordinate": "E3",
                 "type": "control",
                 "control_barcode": "DS000010012",
                 "control_coordinate": "B7",
                 "control": "positive",
+                "picked": True,
+                "created_at": timestamp,
+                "date_picked": timestamp,
+            },
+            {
+                "automation_system_run_id": 1,
+                "destination_coordinate": "E3",
+                "type": "empty",
+                "picked": False,
+                "created_at": timestamp,
+                "date_picked": "",
             },
-            {"automation_system_run_id": "1", "destination_coordinate": "", "type": "empty"},
         ]
 
-        # compare list of dicts
-        pairs = zip(actual, expected)
-        assert any(x != y for x, y in pairs)
+        for sample in actual:
+            assert sample in expected
```

Comment:
Really we should be using [`assertCountEqual`](https://docs.python.org/3.2/library/unittest.html#unittest.TestCase.assertCountEqual) for this.  The test as written is a little loose.  It's quite inefficient and you could have `['a', 'b', 'c', 'c']` in `actual` but only `['a', 'b', 'c']` in `expected` and the test would still pass.

Same bad outcome check further up as well (line 64).